### ISSUE TITLE
Spotlightify doesn't play current song

### ISSUE THREAD

**Issue by WatchTheory** [spotlightify/spotlightify#102]
Windows 10 
Version : 22H2
OS Build : 19045.4894

Recreate problem
- open Spotify 
- open Spotlightify
- Control + Alt + Space pulls up Spotlightify
- Use spotify (not Spotlightify) to play a song in Spotify 
- Using spotlightify to play current song.

**Comment by petergmurphy** [comment]
Hi WatchTheory,

I'm just trying to better understand the problem. Is the problem that Spotlightify isn't able to re-play the song that's currently playing on Spotify? And are you presented with an error after you try and play the song?

Peter

**Comment by WatchTheory** [comment]
Correct, Spotlightify isn't able to re-play the currently song. I tried to do a screen recording of the problem but Spotlightify wasn't visible on the recording

**Comment by petergmurphy** [comment]
If you're able to capture the issue with something like OBS full screen recording that would be great. I haven't had any luck with recreating the issue, I'm able to re-play a song that has been selected through the Spotify client using Spotlightify.

**Comment by WatchTheory** [comment]
Initially, here is what I was doing in Spotlightify. Keep your eye on the play and pause button. 

Control + Alt + Space pulls up Spotlightify typing  "pause" keyword, which pauses the song, then typing "play" which moves me to find a play a song. However, I discovered I had another device connected to my network. So in order to play a song, I have to do (with the song still paused in Spotify) Control + Alt + Space > device , select my computer, and then the song continues playing.  

https://github.com/user-attachments/assets/9df30d64-6f7f-4b49-a267-6bee4cd2a5ff

**Comment by petergmurphy** [comment]
To resume playback of the song there is the "resume" command but I'm realising I'll probably need to change the names/keywords of some of these commands to make them more intuitive to users.

The area of UX is quite new to me but your feedback is super useful, thanks!

**Comment by WatchTheory** [comment]
Glad I can help, I love using spotlightify.
